Bootstraps/BootstrapInterface.php

```diff
@@ -3,10 +3,10 @@
 namespace PHPPM\Bootstraps;
 
 /**
- * All application bootstraps must implement this interface
+ * All application bootstraps must implement this interface.
  */
 interface BootstrapInterface
 {
     public function getApplication();
-    public function getStaticDirectory();
+    public function getStaticDirectory(): string;
```

Comment:
This change forces to use php 7.0 or higher. ``composer.json`` only requires php 5.6 or higher.
This results in a BC-break.

Reply:
There are anyway no releases, yet. There are bigger breaks. And increasing the required PHP version isn't a BC break.

Reply:
I'm certain that not changing `composer.json` to require 7.0 instead of 5.6 was simply an oversight and will be fixed if this PR is merged. Composer will not pull versions that increase the PHP version requirement if at least that version of PHP is not installed.